CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# 🤝 Contributing to octasol
+
+Welcome to Octasol! We are glad about your interest in contributing. Please take a look at this document to support an easy and effective contribution process for everyone.
+
+Following these guidelines shows respect for our maintainer's time, and in turn, they will respond to your issues and evaluate your contributions.
+
+> [!NOTE] 
+> Please submit one Pull Request per issue! 
```

Comment:
This can be removed as we can allow multiple PRs and collaboration on a issue

Reply:
Thanks for the feedback. I pushed a new commit considering your feedback